battlecode-engine/src/world.rs

```diff
@@ -3228,11 +3237,12 @@ mod tests {
         // The worker can replicate to the north.
         assert![world.can_replicate(worker, Direction::North)];
         assert![world.replicate(worker, Direction::North).is_ok()];
-        assert_eq![world.karbonite(), 85];
+        assert_eq![world.karbonite(), 93];
```

Comment:
Huh? Why did this change?

Reply:
I did end_round() so the player also gained karbonite.